src/main/python/pybuilder/plugins/python/coverage_plugin.py

```diff
@@ -39,6 +39,8 @@ def init_coverage_properties(project):
     project.build_depends_on("coverage")
 
     project.set_property_if_unset("coverage_threshold_warn", 70)
+    project.set_property_if_unset("coverage_branch_threshold_warn", 0)
```

Comment:
I'll be setting it to 100 in my projects immediately upon adoption, so it's not an issue for me. 
I suppose the other concern of people not reading the release notes is tangential - if they don't care/didn't care to have branch threshold why bother them with it at all?